resources/js/visual-editor/blocks/file/file.css

```
@@ -0,0 +1,110 @@
+}
+
+.wp-block-file:not(.wp-element-button) {
+    font-size: 0.8em;
+}
+
+.wp-block-file.aligncenter {
+    text-align: center;
+}
+
+.wp-block-file.alignright {
+    /*rtl:ignore*/
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)